demos/demo-minimal-js/index.js

```diff
@@ -191,6 +191,7 @@ export function outgoingCall() {
     cti.outgoingCall({
       createEngagement: true,
       phoneNumber: state.toNumber,
```

Comment:
Could you change `phoneNumber` to `toNumber` is this or a follow up PR?

Reply:
I didn't change it because the issue is in a different Jira ticket: CALL-6250